frontend/src/store/modules/place/mutations.js

```diff
@@ -1,3 +1,11 @@
+import {STATUS_LIKED, STATUS_DISLIKED, STATUS_NONE} from '@/services/api/codes';
+
+function findReviewById (state, reviewId) {
```

Comment:
better to make it like findReviewById(places, reviewId)

Reply:
fixed